### ISSUE TITLE
Polycentric comments are not being posted (Backfill failed)

### ISSUE THREAD

**Issue by TheQuickFox** [futo-org/grayjay-android#1202]
### What happened?

![Polyfail](https://github.com/user-attachments/assets/94a49dbb-3e2f-4829-a40b-a87cd0470a55)

When I make a polycentric comment under a video, it seems to be posted. But when I close the video I get an error message: 
`Backfill failed 1/1`
`Failed to backfill server https://srv1-prod.polycentric.io java.net.SocketTimeoutException: timeout`
`Never Dismiss`

Android 13, Sony Xperia XZ1 Compact
Disabling my ad blocker AdAway does not solve the issue.

It is not dependent on a platform. Both polycentric comments on YouTube and Odysee videos do not appear.
My comments do show up in my polycentric comment history, but is appears as I am shadowbenned because I get no likes, dislikes and they only show up there, not under the videos.

A couple of weeks ago my comments did appear on the same smartphone.

### Grayjay Version

253

### What plugins are you seeing the problem on?

Other

### When do you experience the issue?

- [X] While logged in
- [ ] While logged out
- [ ] N/A

**Comment by lmsalustri** [comment]
Duplicate of #1074

**Comment by johnsmith744** [comment]
I don't get this "backfill" message but my comments are not posted too.  Does show in history.

**Comment by lmsalustri** [comment]
@TheQuickFox Duplicate, please close (Repo is unmoderated)

**Comment by Pyrate37** [comment]
Failures seem intentional. Controversial commenting seems to trigger the backfill fail.

**Comment by TheQuickFox** [comment]
> Failures seem intentional. Controversial commenting seems to trigger the backfill fail.

I don't think this is the issue. All my PolyCentric comments are failing. And all or at least the great majority are not controversial.

**Comment by johnsmith744** [comment]
> > Failures seem intentional. Controversial commenting seems to trigger the backfill fail.
> 
> I don't think this is the issue. All my PolyCentric comments are failing. And all or at least the great majority are not controversial.

'if nobody hate what you say than you are not saying anything' - I forgot who said this.  Though, imo this is half true only.  Because 'if you are not with us, than you are against us and we'll murder you just to be safe' or something like that.  LOL

**Comment by mathix420** [comment]
Potential fix here https://github.com/futo-org/grayjay-android/issues/1074#issuecomment-2352175180

**Comment by TheQuickFox** [comment]
> Potential fix here [#1074 (comment)](https://github.com/futo-org/grayjay-android/issues/1074#issuecomment-2352175180)

Yes, this fixes my issue.
